src/js/components/Chart/index.d.ts

```diff
@@ -1,32 +1,98 @@
-import * as React from "react";
+import * as React from 'react';
 import {
   A11yTitleType,
   AlignSelfType,
   EdgeSizeType,
   GapType,
   GridAreaType,
   MarginType,
-} from "../../utils";
+} from '../../utils';
+
+type ThicknessType =
```

Comment:
It will be nice to export it so TS users could leverage it as well.
And I'm thinking we should move it to utils/index.d.ts.

Reply:
yes, done